### PR TITLE
Remove live streaming and add link glossary to the navbar

### PR DESCRIPTION
**Pull request by agatku**
I was _the muscle_, but _the brain_ of this operation was @bmispelon who is sitting next to me. :)

We don't need live streaming anymore. Also, Baptiste suggested adding a link to a glossary to the navbar.

### CODE REVIEW

**Comment by agatku** [comment]
Work in progress

**Comment by agatku** [comment]
Ok, now it should be ready to merge :)

**Comment by xavierdutreilh** [comment]
Thanks for the cleanup. 👏

File: templates/base.html [hunk 1]
```diff
@@ -29,12 +29,11 @@
                 {% activeurl %}
                 <ul class="row list-inline">
                     <li><a href="{% url 'home' %}" data-emoji-alt="🏡">Home</a></li>
-{#                    <li><a href="https://ti.to/djangocon-europe-2016/budapest/" data-emoji-alt="🎫">Tickets!</a></li>#}
-                                      <li><a href="/live/">Live!</a></li>
                     <li><a href="{% url 'schedule:schedule' %}" data-emoji-alt="🔈🔈🔈">Schedule</a></li>
                     <li><a href="{% url 'venue' %}" data-emoji-alt="🏠🎷🎵">Venue</a></li>
                     <li><a href="{% url 'blog:index' %}" data-emoji-alt="📰✍">Blog</a></li>
                     <li><a href="{% url 'faq:list' %}" data-emoji-alt="🙋🙋🙋">FAQ</a></li>
+                    <li><a href="{% url 'venue' %}" data-emoji-alt="📖">Glossary</a></li>
```
Comment (xavierdutreilh) [inline templates/base.html:36]: Missed this one while reviewing, and before merging. It should have been `url 'glossary'`.

**Comment by agatku** [comment]
Ah, sorry :C I see you already fixed this - thanks! ✨

**Pull request merged**